src/views/createWallet/createWallet.html

```diff
@@ -33,10 +33,10 @@
         >
       </div>
     </v-container>
-
+    
     <v-container v-if="currentTab == 'import'">
       <v-row class="caption" justify="center">
-        <span>You can use this to import a wallet from the Threefold app.</span>
+        <span>You can use this to import a wallet from the Threefold app or load a stellar secret.</span>
```

Comment:
small change: invert this, make stellar secret default and wallet seed the exception